modules/check_in/spec/services/v2/chip/service_spec.rb

```diff
@@ -27,76 +27,34 @@
     let(:faraday_response) { Faraday::Response.new(body: resp, status: 200) }
     let(:hsh) { { data: faraday_response.body, status: faraday_response.status } }
 
-    context 'with service_refactor feature flag on' do
-      before do
-        allow(Flipper).to receive(:enabled?)
-          .with('check_in_experience_chip_service_refactor').and_return(true)
-      end
-
-      context 'when token is already present' do
-        before do
-          allow_any_instance_of(::V2::Chip::Service).to receive(:token).and_return('jwt-token-123-abc')
-          allow_any_instance_of(::V2::Chip::Client).to receive(:check_in_appointment)
-            .and_return(Faraday::Response.new(body: 'Checkin successful', status: 200))
-        end
+    before do
+      allow(Flipper).to receive(:enabled?)
+        .with('check_in_experience_chip_service_refactor').and_return(true)
```

Comment:
Wondering if there is any reason to mock `check_in_experience_chip_service_refactor` feature since we are completely removing as part of this change?

Reply:
good catch, thanks. I missed it, removed in b60e501492e7268317abbceac9c84aa4d2ed89f8